lib/fastlyctl/commands/tls/managed.rb

```diff
@@ -0,0 +1,125 @@
+module FastlyCTL
+  class TLSManagedSubCmd < SubCommandBase
+    SubcommandPrefix = "tls managed"
```

Comment:
Stupid nitpick, but is our new TLS self-service still called "managed"?  We have Free, Platform and Concierge coupled with Fastly managed certs, BYO certs, etc.  The whole thing is a bit confusing still. Point being, i want to make sure you leave yourself enough flexibility here for all the diff TLS offerings we have.  If managed is the right choice, all good, just wanted to sanity check.

Reply:
It's referencing this: https://docs.fastly.com/api/tls-subscriptions

The title on that page is `Fastly-managed TLS subscriptions`. If we are calling it something else I can easily swap the name to whatever it should be.

Reply:
I was planning on having different subcommands for different categories. For instance, `platform` for Platform TLS`, and `managed` for managed certificate subscriptions. Additionally, the other API endpoints can just be called what they are: `private` for private keys, `domain` for domains, `cert` for certificates, and `activation` for activations.

Reply:
Would you rather I called them `subscriptions`? "Subscription" is the noun while "managed" is the modifier, so gramatically it probably does make more sense to call them subscriptions.